docs/StartHere.md

```diff
@@ -0,0 +1,9 @@
+# Tidepool uploader developer guide
+
+As you're getting ready to develop code in this repository, we recommend starting with the following documents:
```

Comment:
That's something to discuss - certainly I think you should have a go at some of the driver-side architecture, code style, etc. documentation.